node/config.go

```diff
@@ -166,6 +166,9 @@ type Config struct {
 	// ephemeral nodes).
 	GRPCPort int `toml:",omitempty"`
 
+	// APINRAppName string `toml:",omitempty"`
+	// APINRLicense string `toml:",omitempty"`
```

Comment:
Will these be used in the upcoming PR, which uses environmental variable to update NewRelic related values?

Reply:
@ehnuje I removed these comments and make the node read os environment variables when NewRelic is configured